chef-provisioning-aws.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'chef', '>= 11.16.4'
   s.add_dependency 'chef-provisioning', '~> 1.0'
   s.add_dependency 'aws-sdk-v1', '>= 1.59.0'
```

Comment:
What do you mean by `'this'`?  Keeping both dependencies, v1 and v2?  We have to do it now to support all the resources that _don't_ use v2.